actions/generate_changelog/default-prompt.txt

```diff
@@ -1,17 +1,25 @@
 You are an expert at analyzing Pull Requests and generating changelog entries for non-technical end-users.
 Analyze the following PR data and generate a user-friendly changelog entry that explains how the changes will benefit them.
 
-Focus on:
+<data>
+PR Data:
+{pr_data}
+</data>
+
+<instructions>
+Use the pull request's commits, linked issues, description and related comments to generate the changelog. Focus on:
 - Explaining changes in plain language that non-technical users will understand
 - Describing new features in terms of what users can now do
 - Explaining fixes in terms of problems that have been solved
 - Mentioning any changes that users need to be aware of
+- Document any actions that may be required by the users
 - Being informative without being overwhelming
 - Avoiding technical jargon, implementation details, and developer-centric information
+</instructions>
 
-Do NOT categorize changes by type (features/fixes/etc). Instead, present all changes in a simple bullet-point list.
-Format your response as markdown with clear, straightforward bullet points.
-Each bullet point should explain one change and its benefit to the user.
-
-PR Data:
-{pr_data}
\ No newline at end of file
+<formatting>
+Ensure the following formatting in the changelog:
+- Format your response as Github-flavoured markdown.
+- Explain the user impact ("Users can now...")
+- Do NOT categorize changes by type (features/fixes/etc). Instead, present all changes in a simple bullet-point list.
```

Comment:
Try "present changes as analytical, thoughtful, journalistic prose without bulleted lists, subheadings or emojis"

Reply:
@agritheory I updated the prompt, and the output from it can be found [here](https://github.com/Alchez/beam/pull/1#issuecomment-2857827924).

PS: If the output needs further tuning, we can also try Anthropic's "improve prompt" feature over at https://console.anthropic.com/dashboard.